src/types/entities/Project.ts

```diff
@@ -8,6 +8,7 @@ export type ProjectId = string;
 export interface Project extends BaseEntity {
   id: ProjectId;
   code: null;
+  color?: string;
```

Comment:
if not present it's `null` since it comes from JSON

I think it should be `color: string | null`